src/index.js

```diff
@@ -1,85 +1,65 @@
 const TESTING = global.COMB !== undefined;
-
-if (!TESTING)
+if (!TESTING) {
   COMB = require('@holo-host/comb').COMB;
+}
 
 const { EventEmitter } = require('events');
 
-function makeUrlAbsolute (url) {
+function makeUrlAbsolute(url) {
   return new URL(url, window.location).href
 }
 
-class Connection extends EventEmitter {
+class WebSDKAPI extends EventEmitter {
 
-  constructor(url, signalCb, opts) {
+  constructor() {
     super();
+    this.child = null;
+    this.available = null
+  }
+
+  ready = () => {
+    return new Promise((resolve, reject) => {
+      this.available = resolve
+    });
+  }
 
+  connect = async ({ chaperoneUrl, authFormCustomization } = {}) => {
```

Comment:
a constructor can't be async

Reply:
but I think the connect function should be static.

```suggestion
  static connect = async ({ chaperoneUrl, authFormCustomization } = {}) => {
```

That's how it is in the example code. So instead of saying 

```
const websdk = new WebSdk()
const client = await websdk.connect()
```

you can just say

```
const client = await WebSdk.connect()
```

@JettTech - any reason not to do it this way?

Reply:
I was leaving it as a instantiated class originally just to have a local copy of the websdk api, which is needed to call the expected holo endpoints.  However, I like the idea of making this a static method, so I've updated it as static and am simply returning an instantiation of the `WebSdkApi` type from the`connect` itself.